diff --git a/LiveWallpaper/Components/ProgressRing.swift b/LiveWallpaper/Components/ProgressRing.swift
index 4c38f23..a4a71fa 100644
--- a/LiveWallpaper/Components/ProgressRing.swift
+++ b/LiveWallpaper/Components/ProgressRing.swift
@@ -26,3 +26,5 @@
+#if DEBUG
 #Preview {
     ProgressRing(progress: 0.8, size: 100)
 }
+#endif
diff --git a/LiveWallpaper/ContentView.swift b/LiveWallpaper/ContentView.swift
index dc19eaf..0f3fb9a 100644
--- a/LiveWallpaper/ContentView.swift
+++ b/LiveWallpaper/ContentView.swift
@@ -65,3 +65,5 @@
+#if DEBUG
 #Preview {
     ContentView()
 }
+#endif
diff --git a/LiveWallpaper/Views/Recent.swift b/LiveWallpaper/Views/Recent.swift
index 68c1b62..d05bcb7 100644
--- a/LiveWallpaper/Views/Recent.swift
+++ b/LiveWallpaper/Views/Recent.swift
@@ -75,3 +75,5 @@
+#if DEBUG
 #Preview {
     Recent()
 }
+#endif
